session/src/main/java/org/apache/iotdb/session/pool/SessionPool.java

```diff
@@ -108,6 +111,20 @@ public SessionPool(String host, int port, String user, String password, int maxS
         Config.DEFAULT_CONNECTION_TIMEOUT_MS);
   }
 
+  public SessionPool(List<String> nodeUrls, String user, String password, int maxSize) {
```

Comment:
Please update the sessionPool documentation to show greater fault tolerance when passing multiple endpoints like a session

Reply:
Done